electron_app/src/spell-check.js

```diff
@@ -0,0 +1,217 @@
+/*
+Copyright 2018 New Vector Ltd
```

Comment:
We can't take copyright for this. It looks almost identical to Rocket.Chat's implementation (https://github.com/RocketChat/Rocket.Chat.Electron/blob/7ad28f3f24f7b32b22ea75ed4e35ed2b173b933c/src/preload/spellchecking.js) which is under MIT. We need to give them credit, and use their license for this file, in order to include it.

Reply:
we could change something inside. For example their implementation allow Multilanguage, our not ;)
Please suggest something

Reply:
I'm not a lawyer, but I believe we need to have the file under MIT and their copyright. Then when changes are made the applicable party's copyright is added underneath the Rocket.Chat copyright.

Reply:
Please take a look at new version